lighthouse/terraform/secrets.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_secretsmanager_secret" "db_username" {
+  name = "ci/lighthouse/database/username"
```

Comment:
Why do we say lighthouse is `ci`?

Reply:
That's what it calls itself https://github.com/GoogleChrome/lighthouse-ci

Reply:
Ah - yea, so that's "Lighthouse that can run in CI". I'm not sure we have a standard to how we namespace our things - but I would expect something prefixed with `ci` to be something that helps us run our CI rather than runs in our CI (which is loads).

Would this maybe go under `monitoring` or maybe even just `lighthouse`?

Reply:
yeah probably just `lighthouse` works